CMakeLists.txt

```diff
@@ -110,6 +110,11 @@ endif()
 
 # Android options
 if(SFML_OS_ANDROID)
+    sfml_set_option(SFML_ANDROID_USE_SUSPEND_AWARE_CLOCK FALSE BOOL "TRUE to use an sf::Clock implementation which takes system sleep time into account, FALSE to default to another available monotonic clock")
```

Comment:
"To take into account" is almost a synonym for "to be aware". What is still not unambiguously clear, is whether that means that the clock _stops_ during system sleep time (I would use the term "suspend" here), or keeps running. I would really be as explicit as possible. Maybe something like:

> TRUE to use an sf::Clock implementation which pauses while the activity is suspended, FALSE for a monotonic clock that keeps running.

or the other way around, if I got this wrong.